lib/oppen/print_stack.rb

```diff
@@ -7,10 +7,10 @@ module Oppen
   class PrintStack
     # Class that represents an item in the print stack.
     class PrintStackEntry
-      # @return [Integer] Indentation level.
-      attr_reader :offset
       # @return [Token::BreakType] (Called break in the original paper).
       attr_reader :break_type
+      # @return [Integer] Indentation level.
+      attr_reader :offset
```

Comment:
Sometimes we add a blank line between `attr_*`, sometimes we don't. I say we space them out, since we're documenting. Please apply everywhere.